VERSION

```diff
@@ -104,15 +104,15 @@ libompitrace_so_version=60:0:20
 # components-don't-affect-the-build-system abstraction.
 
 # OMPI layer
-libmca_ompi_common_ompio_so_version=60:1:19
+libmca_ompi_common_ompio_so_version=60:2:19
 libmca_ompi_common_monitoring_so_version=60:0:10
 
 # ORTE layer
-libmca_orte_common_alps_so_version=60:0:20
+libmca_orte_common_alps_so_version=60:1:20
```

Comment:
I don't see any changes to files for common_alps_so.  I think it should stay at rev=0

Reply:
I did see this change, so I though I'd bump the age.  Perhaps this doesn't warrant an age bump?
```
diff --git a/orte/mca/ess/alps/ess_alps_module.c b/orte/mca/ess/alps/ess_alps_module.c
index 533a054e..bfbbc3d9 100644
--- a/orte/mca/ess/alps/ess_alps_module.c
+++ b/orte/mca/ess/alps/ess_alps_module.c
@@ -45,7 +45,7 @@ static int rte_finalize(void);
 orte_ess_base_module_t orte_ess_alps_module = {
     rte_init,
     rte_finalize,
-    orte_ess_base_app_abort,
+    NULL,
```

Reply:
Not usually as it isn't an interface in the common/alps library

Reply:
Thanks, I figured that out after looking at it again.